### ISSUE TITLE
Refactor network area and team memberships

### ISSUE THREAD

**Issue by timobrembeck** [digitalfabrik/integreat-cms#2451]
### Motivation

When I designed the access control system, I thought app team users might want to have read-only access to the entire CMS, also to debug specific app-related problems in other regions.
However, in reality they need write-access, but only to the testumgebung and not to other regions.
This causes some confusion when new app team members are created in the CMS.

### Proposed Solution

The simplest solution would be to just remove the app team from the possible team choices and to rename the checkbox "Team member" to "Access to network area" to point out that it is possible to create team members who do not have access to the network area.
Then, app team members would just be created as managers of the testumgebung.

### Alternatives

A more complex solution would be to keep the current app team with the network access, but make all permissions read-only again and somehow make it possible that they have write access for the testumgebung, despite having only read access to other regions.
This could even be used to allow them to edit the region form (in the network area) for "testumgebung", but not for the other regions.

**Comment by osmers** [comment]
@PeterNerlich @MizukiTemma with these acceptance criteria, is it still effort high? If no, pls remove it :)

**Comment by MizukiTemma** [comment]
> [@PeterNerlich](https://github.com/PeterNerlich) [@MizukiTemma](https://github.com/MizukiTemma) with these acceptance criteria, is it still effort high? If no, pls remove it :)

It's effort: low with that solution. Label changed.

**Comment by MizukiTemma** [comment]
The topic was raised again in a chat: https://chat.tuerantuer.org/digitalfabrik/pl/uigy6n3xmbyqbn9xi6nfc9pjma

**Comment by seluianova** [comment]
> Existing assignments to the App Team changed to role admin in Testumgebung.

it means to role "Manager", right?
or it means this checkbox, meaning "Experienced user"?

<img width="131" height="227" alt="Image" src="https://github.com/user-attachments/assets/351cc872-4867-4ff7-ab12-8be88c88fd8b" />

_btw is it intended that the "experienced user" checkbox is displayed as "administrator" in the user table?_

<img width="565" height="126" alt="Image" src="https://github.com/user-attachments/assets/839e1b52-8273-4629-a569-d6a7e308a6a3" />

**Comment by osmers** [comment]
@seluianova yes, sorry, it means the role manager, in German "Verwalter:in". 
Administrator should not come from the experienced user checkbox. I think it comes from the team membership - so as part of the CMS team, you are admin here as well. 
Quite a lot of people who are experienced users, as per the checkbox, so not have the tick under Administrator.

**Comment by osmers** [comment]
I think I already changed all app-team members to be only managers.

**Comment by seluianova** [comment]
> I think I already changed all app-team members to be only managers.

ah, so I don't have to include this into migration...

> Existing assignments to the App Team changed to role admin in Testumgebung.

**Comment by seluianova** [comment]
> Administrator should not come from the experienced user checkbox. I think it comes from the team membership - so as part of the CMS team, you are admin here as well.

@osmers it comes from the "Superuser" checkbox. So "Administrator" means "Superuser". Sorry for the confusion, I was a bit confused myself 🙈 
"Team membership" does not automatically mean "Administrator", it is separate column:

<img width="1197" height="440" alt="Image" src="https://github.com/user-attachments/assets/2a314a4d-d6e9-439f-8d51-f3234660aa60" />

so, is it correct that the "Superuser" checkbox displayed as "Administrator"?

**Comment by osmers** [comment]
> [@osmers](https://github.com/osmers) it comes from the "Superuser" checkbox. So "Administrator" means "Superuser". Sorry for the confusion, I was a bit confused myself 🙈
> "Team membership" does not automatically mean "Administrator", it is separate column:

True -  I can also see in your screenshot that team membership of a specific team does not signify admin status. I cannot see the superuser checkbox :D maybe that is only visible to the CMS team.

As to whether this is understandable or correct: I'd say not very understandable, but I'm not sure if naming it super-user will help, especially since only very few people can see the check box

**Comment by osmers** [comment]
> > I think I already changed all app-team members to be only managers.
> 
> ah, so I don't have to include this into migration...
> 
> > Existing assignments to the App Team changed to role admin in Testumgebung.

Yes - you can double check, but there should be no assignments to app team left in the user database: https://admin.integreat-app.de/users/?search_query=&role=APP_TEAM&permissions=&region=

**Comment by MizukiTemma** [comment]
@osmers @seluianova 

> The simplest solution would be ...... to rename the checkbox "Team member" to "Access to network area" to point out that it is possible to create team members who do not have access to the network area.

> (A tooltip or help text explains that team members can also be created without access to the network area.)


In the CMS GUI it's not possible to create a team member without access to the network area, the team choice (CMS, Service, Marketing) first appears when "Access to the network area" is clicked.

But that does not affect the implementation and functionalities, and the new label is more intuitive, so I just approved the PR without tooltip or something similar stating it.

**Comment by juliankehne** [comment]
Tested on cms-test:

Only 1 cosmetic issue I found, but its very minor so I moved it to Done
https://github.com/digitalfabrik/integreat-cms/issues/4288
